README.md

```diff
@@ -3,14 +3,40 @@
 
 `pshtt` was developed to _push_ organizations— especially large ones like the US Federal Government :us: — to adopt HTTPS across the enterprise. Federal .gov domains must comply with [M-15-13](https://https.cio.gov), a 2015 memorandum from the White House Office of Management and Budget that requires federal agencies to enforce HTTPS on their web sites and services by the end of 2016. Much has been done, and [still more yet to do](https://18f.gsa.gov/2017/01/04/tracking-the-us-governments-progress-on-moving-https/).
 
-`pshtt` is a collaboration between GSA's 18F and the DHS National Cybersecurity Assessments and Technical Services team.
+`pshtt` is a collaboration between the [Department of Homeland Security's National Cybersecurity Assessments and Technical Services (NCATS) team](https://github.com/dhs-ncats) and [the General Service Administration's 18F team](https://18f.gsa.gov), with [contributions from NASA and various non-governmental organizations](https://github.com/dhs-ncats/pshtt/graphs/contributors).
```

Comment:
@h-m-f-t @egyptiankarim This is a separate topic from the main PR, and I can separate it if need be, but while I was editing the README I updated the collaboration text to link to things, order NCATS first, and to mention NASA's contributions.